src/asciidoc/reference/customVcc/customVccReference.adoc

```diff
@@ -152,7 +152,7 @@ However, this object can be useful when creating more complex custom controls, l
 
```

Comment:
In Interfaces > Props > Properties, there's an extra underscore before the type of `config`, which should be `_config`. We probably need to escape the first underscore (and no one noticed it in previous drafts).

Reply:
Removed the '_' before `config`.

Reply:
In the package code, the property name is `_config`, so we might need that character. If so, we can escape it so it doesn't mess with the Asciidoc markup. E.g. 
```
`+_config+` – _Object_, 
```

Reply:
Putting back the '_'. I don't understand your original comment then about an extra underscore.